src/js/media-controller.js

```diff
@@ -159,14 +159,63 @@ class MediaController extends MediaContainer {
       MEDIA_ENTER_PIP_REQUEST: () => {
         const media = this.media;
 
-        if (!document.pictureInPictureEnabled) return;
+        if (!document.pictureInPictureEnabled) {
+          console.warn('MediaChrome: Picture-in-picture is not enabled');
```

Comment:
nitpick(non-blocking): Per prior comment, we should consider move these inline warning strings to constants to anticipate i18n.

Reply:
Is i18n needed/realistic for dev-facing console messages or is that more for future user-facing messages?